tests/test_app/templates/cell.twig

```diff
@@ -0,0 +1 @@
+{{ cell('Test', data=[10]) }}
```

Comment:
Can't we just use `{{ cell('Test', [10]) }}`?

Reply:
I don't see the need to change it as it's clear, but yes data works the same as the first argument.

Reply:
Why would any type extra when simply using `[0]` instead of `data=[10]` works the same?

Reply:
I was just testing it. I don't see why this is a big issue.

Reply:
For me test cases are also usage docs :slightly_smiling_face:. Someone reading the tests might think the `data=` is actually necessary to pass the data to cell.

Reply:
Sure, just for you.